docs/topics/api/addons.rst

```diff
@@ -60,15 +60,17 @@ This endpoint allows you to search through public add-ons.
     ==============  ==========================================================
          Parameter  Description
     ==============  ==========================================================
-           created  Creation date, descending
-         downloads  Number of weekly downloads, descending
+           created  Creation date, descending.
+         downloads  Number of weekly downloads, descending.
            hotness  Hotness (average number of users progression), descending.
             rating  Bayesian rating, descending.
-           updated  Last updated date, descending
+         relevance  Search query relevance, descending.
```

Comment:
Putting "default" or "default (with search query)" would be handy for those (like me) who quickly scan a table for the default. Maybe bolding the defaults?